src/Symfony/Component/Messenger/Transport/Schedule/ScheduleConfig.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Schedule;
+
+use Symfony\Component\Messenger\Transport\Schedule\Trigger\TriggerInterface;
+
+final class ScheduleConfig
+{
+    /**
+     * @var array<int, array{TriggerInterface, object}>
+     */
+    private array $schedule = [];
+
+    /**
+     * @param iterable<array{TriggerInterface, object}> $schedule
+     */
+    public function __construct(iterable $schedule = [])
```

Comment:
```suggestion
    public function __construct(
        private iterable $schedule = [],
    )
```

Reply:
It won't work like that. The "itarable" is iterated and add() method is called for types checking.